### PR TITLE
Reorganized docs a little bit; Added more info about analytics, billi…

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: content/Services/RHOBS/tenant-groups/telemeter.md [hunk 1]
```diff
@@ -8,8 +8,13 @@ Telemeter is the metrics-only hard tenant of the RHOBS service designed as a cen
...
 
-* Blair Rampling
-* Christian Heidenreich
+* Shannon Wilber
+
+## Big Picture Overview
+
+![](../../../assets/telemeter.png)
+
+*[Source](https://docs.google.com/drawings/d/1eIAxCUS2v8Bt0-Ken2gHnx-Q1u8JNPfi2rMB-Azz5zI/edit)*
```
Comment (moadz) [inline content/Services/RHOBS/tenant-groups/telemeter.md:17]: @matej-g the diagram we were wishing for

File: content/Services/RHOBS/analytics.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Analytics based on Observability Data
+
+It is possible to use analytic tool that works with [`Apache Parquet`](https://parquet.apache.org/) format (e.g [Amazon RedShift](https://aws.amazon.com/redshift/)) on data stored in RHOBS.
```
Comment (moadz) [inline content/Services/RHOBS/analytics.md:3]:
```suggestion
We use the [`Apache Parquet`](https://parquet.apache.org/) format (e.g [Amazon RedShift](https://aws.amazon.com/redshift/)) to persist metrics data for RHOBS analytics. 
```

Passive voice -> active

File: content/Services/RHOBS/analytics.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+DataHub and CCX teams are currently leveraging such analytics pipeline based on [Telemeter](tenant-groups/telemeter.md) from group. The pipeline looks as follows:
+
+![](../../assets/telemeter-analytics.png)
+
+*[Source](https://docs.google.com/drawings/d/19Z0vtVjlvU_p6aU3hn6PXdN0AWbd2FA4RAC0dT3hU_w/edit)*
+
+1. DataHub runs [Thanos Replicate](https://thanos.io/tip/components/tools.md/#bucket-replicate) tool in their PSI cluster, which copies all fresh 2h size blocks from RHOBS, once they appear in RHOBS S3 object storage (~3h after ingestion).
+2. Replicated blocks are copied to DataHub Ceph object storage and stored for unlimited time.
+3. DataHub runs smaller Thanos system. For this part of the system only the Thanos Store Gateway is needed to provide Store API for further Obslytics step.
+4. Instance of [Obslytics](https://github.com/thanos-community/obslytics) is then transforming Thanos metrics format to [`Apache Parquet`](https://parquet.apache.org/) for desired metrics and it saves it in Ceph Object Storage again.
```
Comment (moadz) [inline content/Services/RHOBS/analytics.md:22]:
```suggestion
4. Instance of [Obslytics](https://github.com/thanos-community/obslytics) transforms Thanos metrics format to [`Apache Parquet`](https://parquet.apache.org/) and saves it in Ceph Object Storage.
```

File: content/Services/RHOBS/analytics.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+5. Parquet data is then imported to [Amazon RedShift](https://aws.amazon.com/redshift/), so it can be joined with other data sources.
```
Comment (moadz) [inline content/Services/RHOBS/analytics.md:23]:
```suggestion
5. Parquet data is then imported in [Amazon RedShift](https://aws.amazon.com/redshift/), and is joined with other data sources.
```

File: content/Services/RHOBS/tenant-groups/README.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# Tenant Groups
+
+From business perspective we can divide RHOBS into a few main "Tenant Groups" that can hold multiple soft tenants.
```
Comment (moadz) [inline content/Services/RHOBS/tenant-groups/README.md:3]:
```suggestion
From a business perspective we can divide RHOBS into a few main "Tenant Groups" that can hold multiple soft tenants.
```

File: content/Services/RHOBS/tenant-groups/README.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+
+The main difference is currently in SLOs.
```
Comment (moadz) [inline content/Services/RHOBS/tenant-groups/README.md:5]: The main difference of what?

**Review by moadz — APPROVED** [review]
LGTM % some passive voice => active voice suggestions

File: content/Services/RHOBS/consumption-billing.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# Consumption Billing
+
+Consumption billing is done by Red Hat SubWatch. Data is periodically taken from RHOBS metric Query APIs. If you are internal Red Hat Manage Service, contact SubWatch team on how you can use consumption billing.
```
Comment (matej-g) [inline content/Services/RHOBS/consumption-billing.md:3]:
```suggestion
Consumption billing is done by Red Hat SubWatch. Data is periodically taken from RHOBS metrics Query APIs. If you are internal Red Hat Managed Service, contact SubWatch team to learn how you can use consumption billing.
```

File: content/Services/RHOBS/tenant-groups/README.md (hunk 5 above)
Comment (matej-g) [reply content/Services/RHOBS/tenant-groups/README.md:3]: And what are the tenant groups? :thinking:

**Review by matej-g — COMMENTED** [review]
Looks good :+1: Few nits

File: content/Services/RHOBS/tenant-groups/README.md (hunk 5 above)
Comment (bwplotka) [reply content/Services/RHOBS/tenant-groups/README.md:3]: Two instance: Telemeter and MST. How can we name it differently?

Comment (moadz) [reply content/Services/RHOBS/tenant-groups/README.md:3]: Are tenant groups an implementation-detail or just a term we're using for describing mutually exclusive tenants across different Observatorium deployments? If the former then keep, if the latter then we shouldn't really use them as it's tenant is an already overloaded term.

File: content/Services/RHOBS/analytics.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
+# Analytics based on Observability Data
+
+It is possible to use analytic tool that works with [`Apache Parquet`](https://parquet.apache.org/) format (e.g [Amazon RedShift](https://aws.amazon.com/redshift/)) on data stored in RHOBS.
+
+Red Hat maintains community-driven project that is able to transform Prometheus data into Parquet files. See [Obslytics project to learn more](https://github.com/thanos-community/obslytics). It works by consuming (currently internal) RHOBS StoreAPI.
+
+If you wish to use this tool against RHOBS, contact us to discuss this feature request or add ticket to our `MON` JIRA board.
```
[the remark this answers is not in the document]
Comment (jessicalins) [reply content/Services/RHOBS/analytics.md:7]: I'd add also the link https://issues.redhat.com/projects/MON/summary

File: content/Services/RHOBS/analytics.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
+# Analytics based on Observability Data
+
+It is possible to use analytic tool that works with [`Apache Parquet`](https://parquet.apache.org/) format (e.g [Amazon RedShift](https://aws.amazon.com/redshift/)) on data stored in RHOBS.
+
+Red Hat maintains community-driven project that is able to transform Prometheus data into Parquet files. See [Obslytics project to learn more](https://github.com/thanos-community/obslytics). It works by consuming (currently internal) RHOBS StoreAPI.
```
[the remark this answers is not in the document]
Comment (jessicalins) [reply content/Services/RHOBS/analytics.md:5]:
```suggestion
Red Hat maintains a community-driven project that is able to transform Prometheus data into Parquet files. See [Obslytics project to learn more](https://github.com/thanos-community/obslytics). It works by consuming (currently internal) RHOBS StoreAPI.
```

File: content/Services/RHOBS/analytics.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+
+If you wish to use this tool against RHOBS, contact us to discuss this feature request or add ticket to our `MON` JIRA board.
+
+## Existing Production Pipelines
+
+### Telemeter Analytics
+
+DataHub and CCX teams are currently leveraging such analytics pipeline based on [Telemeter](tenant-groups/telemeter.md) from group. The pipeline looks as follows:
```
Comment (jessicalins) [inline content/Services/RHOBS/analytics.md:13]: I think we can leave out `from group` ?

**Review by jessicalins — COMMENTED** [review]
looks good - besides the other comments, I left few small suggestions too

File: content/Projects/Observability/observatorium.md [hunk 11]
```diff
@@ -16,7 +16,7 @@ TBD(https://github.com/rhobs/handbook/issues/22)
 * GET /api/metrics/v1/api/v1/query_range
 * GET /api/metrics/v1/api/v1/series
 * GET /api/metrics/v1/api/v1/labels
-* GET /api/metrics/v1/api/v1/label/<name>/values
+* GET /api/metrics/v1/api/v1/label/ <name> /values
```
Comment (matej-g) [inline content/Projects/Observability/observatorium.md:19]: Why the spaces? :thinking: 
```suggestion
* GET /api/metrics/v1/api/v1/label/<name>/values
```

File: content/Services/RHOBS/use-cases/observability.md [hunk 12]
```diff
@@ -16,7 +16,7 @@ TBD(https://github.com/rhobs/handbook/issues/23)
 
 ![SLO](../../../assets/slo-def.png)
 
-If you manage [“Observatorium”](../observatorium.md), the Service Level Objectives can go ultra-high in all dimensions, such as availability and data loss. The freshness aspect for reading APIs is trickier as it also depends on client collection pipeline availability, among other things, which is out of the scope of the Observatorium.
+If you manage [“Observatorium”](../../../Projects/Observability/observatorium.md), the Service Level Objectives can go ultra-high in all dimensions, such as availability and data loss. The freshness aspect for reading APIs is trickier as it also depends on client collection pipeline availability, among other things, which is out of the scope of the Observatorium.
```
Comment (matej-g) [inline content/Services/RHOBS/use-cases/observability.md:19]:
```suggestion
If you manage [“Observatorium”](../../../Projects/Observability/observatorium.md), the Service Level Objectives can go ultra-high in all dimensions, such as availability and data loss. The freshness aspect for reading APIs is trickier, as it also depends on client collection pipeline availability, among other things, which is out of the scope of the Observatorium.
```

**Review by matej-g — APPROVED** [review]
Looks good :+1:

File: content/Projects/Observability/observatorium.md (hunk 11 above)
Comment (bwplotka) [reply content/Projects/Observability/observatorium.md:19]: auto-formatting

**Pull request merged**